backend/src/schema/resolvers/registration.spec.js

```diff
@@ -359,6 +361,7 @@ describe('Signup', () => {
               await expect(neode.all('EmailAddress')).resolves.toHaveLength(2)
               await expect(mutate({ mutation, variables })).resolves.toMatchObject({
                 data: { Signup: { email: 'someuser@example.org' } },
+                errors: undefined,
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-09-30T09:00:02Z" title="Monday, September 30th 2019, 11:00:02 am +02:00">Sep 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -359,6 +361,7 @@ describe('Signup', () => {
               await expect(neode.all('EmailAddress')).resolves.toHaveLength(2)
               await expect(mutate({ mutation, variables })).resolves.toMatchObject({
                 data: { Signup: { email: 'someuser@example.org' } },
+                errors: undefined,
```


and again ☝️